.github/actions/build-ab/index.js

```diff
@@ -133,3 +134,68 @@ if (['dev', 'staging'].includes(args.environment)) {
     }
   }
 }
+
+// add entitlements NG script shim
+(async () => {
```

Comment:
You do not need to wrap this in an `iife`. For github actions, you can assume a global `async` is present.

Reply:
cool didnt know that